assets/js/modules/analytics/common/timzezone-select.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * TimezoneSelect component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Select,
+	Option,
+} from '../../../material-components';
+import { countries } from './countries';
+import classnames from 'classnames';
+
+const { default: { country: allCountries } } = countries;
+
+const TimezoneSelect = ( { timezone, setTimezone, validationIssues, setValidationIssues } ) => {
```

Comment:
Same concern as above here, let's remove the `setValidationIssues` prop and use a `hasError` prop instead of the more complex `validationIssues` one.

Reply:
All fixed in https://github.com/google/site-kit-wp/pull/1389/commits/527201e0a64fca7b4eae50a402147fe29fd0417f